Library/Homebrew/bottle_version.rb

```diff
@@ -63,3 +65,28 @@ def self._parse spec
     super
   end
 end
+
+class BottleVersion
+  def self.parse(path)
+    path = Pathname.new(path) unless path.is_a? Pathname
+    version = BottleVersionParser.parse(path)
+    basename = path.basename.to_s
+    formula_name, _, suffix = basename.rpartition("-#{version}")
+    revision = suffix.match(/_(\d+)/).to_a[1].to_i
+    bottle_revision = suffix.match(/bottle\.(\d+)\.tar\.gz/).to_a[1].to_i
+    new(formula_name, version, revision, bottle_revision)
+  end
+
+  attr_reader :formula_name, :version, :revision, :bottle_revision
+
+  def initialize(formula_name, version, revision, bottle_revision)
+    @formula_name = formula_name
+    @version = version
+    @revision = revision
+    @bottle_revision = bottle_revision
+  end
+
+  def pkg_version
+    PkgVersion.new(@version, @revision)
+  end
```

Comment:
Does it make sense to have both this and `version`? Is there a context in which knowing the version independent of the revision is useful? I don't see `version` being called anywhere, except in the test cases, where it should always be the same as `pkg_version` anyway.

Since the bottle version is always the same as the package version, maybe this should just be called `version` (in the same way it's `Keg#version` rather than `Keg#pkg_version`?